salt/_pillar/metalk8s.py

```diff
@@ -78,12 +79,34 @@ def _load_apiserver(config_data):
     return result
 
 
+def _load_iso_path(config):
+    """Load iso path from BootstrapConfiguration
+
+    """
+    res = {}
+
+    for product in config['products']['metalk8s']:
+        if __salt__['file.directory_exists'](product):
+            version = __salt__['metalk8s.product_version_from_tree'](product)
+        else:
+            version = __salt__['metalk8s.product_version_from_iso'](product)
```

Comment:
Ok we can't do that in ext_pillar as we need the iso to be accessible from the master if we want to do this

Reply:
Only retrieve list of iso/directory in ext_pillar, use them in a custom module